packages/react/src/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { Children, forwardRef, isValidElement } from 'react'
+import type { ForwardedRef, ForwardRefExoticComponent, HTMLAttributes, PropsWithChildren, RefAttributes } from 'react'
+import { DescriptionListDetails } from './DescriptionListDetails'
+import { DescriptionListRow } from './DescriptionListRow'
+import { DescriptionListTerm } from './DescriptionListTerm'
+
+export type DescriptionListProps = PropsWithChildren<HTMLAttributes<HTMLDListElement>> & {
+  inverseColor?: boolean
+}
+
+type DescriptionListComponent = {
+  Term: typeof DescriptionListTerm
+  Details: typeof DescriptionListDetails
+  Row: typeof DescriptionListRow
+} & ForwardRefExoticComponent<DescriptionListProps & RefAttributes<HTMLDListElement>>
+
+export const DescriptionList = forwardRef(
+  ({ children, className, inverseColor, ...restProps }: DescriptionListProps, ref: ForwardedRef<HTMLDListElement>) => {
+    const hasDescriptionListRow = Children.toArray(children).some(
```

Comment:
Tried to avoid using a prop but it seems to not always return true in time for the render. Any react suggestions are welcome.

Reply:
[This](https://react.dev/reference/react/Children#alternatives) suggests using props